core/src/test/java/apoc/create/CreateTest.java

```diff
@@ -40,6 +41,11 @@ public class CreateTest {
         TestUtil.registerProcedure(db,Create.class, Paths.class);
     }
 
+    @After
```

Comment:
should this be AfterClass instead as we don't create/start the db per test method. and doing it per method would explode the test runtime.